### PR TITLE
Update base image to alpine:3.15.4

### PR DESCRIPTION
**Pull request by e0ne**
This patch fixes following CVEs:

* https://nvd.nist.gov/vuln/detail/CVE-2022-0778
* https://nvd.nist.gov/vuln/detail/CVE-2018-25032
* https://nvd.nist.gov/vuln/detail/CVE-2022-0778

Signed-off-by: Ivan Kolodiazhny <ikolodiazhny@nvidia.com>

### CODE REVIEW

**Comment by rollandf** [comment]
LGTM

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-ARG BASE_IMAGE=alpine:3.14.3
+ARG BASE_IMAGE=alpine:3.15.4
```
Comment (adrianchiris) [inline Dockerfile:1]: shall we just use alpine:3.15 ?

Comment (adrianchiris) [reply Dockerfile:1]: so subminor updates come automatically

Comment (e0ne) [reply Dockerfile:1]: I prefer to stick to exact  version to have the same image every build

**Review by adrianchiris — APPROVED** [review]

**Pull request merged**